src/mutation/RelayMutation.js

```diff
@@ -317,7 +342,7 @@ class RelayMutation<Tp: Object> {
         );
         var dataID = RelayFragmentPointer.getDataID(propValue, fragment);
         if (dataID) {
-          resolvedProps[fragmentName] = RelayStore.read(fragment, dataID);
+          resolvedProps[fragmentName] = this._context.read(fragment, dataID);
```

Comment:
@josephsavona, Should I replace `RelayContext` type here with the new `RelayContextInterface`, and add `read` and `readAll` to its definition?